src/Indexer.php

```diff
@@ -104,8 +112,8 @@ public function __construct(
     public function index(): Promise
     {
         return coroutine(function () {
-
-            $pattern = Path::makeAbsolute('**/*.php', $this->rootPath);
+            $fileTypes = implode(',', $this->options->fileTypes);
+            $pattern = Path::makeAbsolute('**/*{' . $fileTypes . '}', $this->rootPath);
```

Comment:
maybe check first if the extensions are still valid?

Reply:
The Glob library should throw an exception for this, so that should not be needed.